### ISSUE TITLE
UI freezes when skipping tracks

### ISSUE THREAD

**Issue by caseif** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1776]
**OS:** Windows 10 Update 1607

**Issue Description:** The UI completely freezes when skipping from one track to the next by any mechanism, either automatic or manual. Media keys also fail to function. This occurs when the player is launched with the `--disable-gpu` option as well (although I'm not sure whether that has an effect on Windows). The only solution is to kill the program via task manager.

**Steps to Reproduce:** Launch the player, skip a song.

**Comment by MiguelCTorres** [comment]
This was happened to me with 3.6 version, today I upgraded to 4.0 and all is working perfect.

Try to update GPMDP 👍

**Comment by MarshallOfSound** [comment]
I can't replicate this issue. Can you download and run these commands from Command Prompt and tell me the output.

```bash
set ELECTRON_ENABLE_LOGGING=true
set ELECTRON_ENABLE_STACK_DUMPING=true
%LOCALAPPDATA%\GPMDP_3\app-4.0.0\Google%20Play%20Music%20Desktop%20Player.exe --dev
```

**Comment by caseif** [comment]
@MiguelCTorres I've reproduced it on both versions.

@MarshallOfSound It seems to have spontaneously resolved itself since this morning, after declaring it was loading the music library after the most recent launch. (The computer wasn't restarted or otherwise suspended between then and now, although a restart earlier today didn't resolve it) Not sure if this was caused by a remote issue on Google's end, but I imagine this can be closed for now (I'll leave that decision to you). I'll reopen if it occurs again.
